packages/Radio/src/components/Group/Group.js

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { ShirtSizes } from "@paprika/helpers/lib/customPropTypes";
+
+const propTypes = {
+  /** aria-labelledby prop on the containing group element */
+  a11yLabelledByText: PropTypes.string,
+
+  /** Are all radios disabled */
+  isDisabled: PropTypes.bool,
+
+  /** Function used to evaluate which radio is selected by default. */
+  defaultCheck: PropTypes.func,
+
+  /** Can deselect any radio */
+  canDeselect: PropTypes.bool,
+
+  /** On change of radio selection. */
+  onChange: PropTypes.func.isRequired,
+
+  /** The individual radio itemss. */
+  children: PropTypes.node,
+
+  /** The size for all radio components. */
+  size: PropTypes.oneOf(ShirtSizes.DEFAULT),
+};
+
+const defaultProps = {
+  a11yLabelledByText: "",
+  canDeselect: false,
+  children: null,
+  defaultCheck: () => {},
+  isDisabled: false,
+  size: ShirtSizes.MEDIUM,
+};
+
+const Group = ({ a11yLabelledByText, canDeselect, children, defaultCheck, ...moreGroupProps }) => {
```

Comment:
please switch this to a form a function instead of arrow function and destructured the properties on the body.

function Group(props) {
  const {a11yLabelledByText, canDeselect, children, defaultCheck, ...moreGroupProps} = props;
}

Reply:
ok out of interest what is the reason for the preference?